main.go

```diff
@@ -734,10 +740,49 @@ func commentToNotifyUser(args commentArgs) error {
 	if !args.mergeConflicts {
 		commentBody = fmt.Sprintf("@%s I have created a PR for you, ready to merge as soon as tests are passed", args.userName)
 	} else {
-		msg := "@%s I have created a PR for you. Unfortunately there were some merge conflicts " +
-			"which I failed to resolve automatically. Therefore the branch of this PR " +
-			"is ready to be merged locally, and then pushed here :)"
-		commentBody = fmt.Sprintf(msg, args.userName)
+		tmplString := `\
+@{{.UserName}} I have created a PR for you.
+Unfortunately, a merge conflict was detected. This means that the conflict will have to be resolved manually by you, human. Then pushed to the PR-branch, once all conflicts are resolved.
+This can be done by following this recipe:
+
+* Make sure that the 'mender-test-bot', and 'open-source' remotes are present in your repository, if not:
+- git remote add mender-test-bot git@github.com:mender-test-bot/{{.Repo}}-enterprise.git
+- git remote add open-source     git@github.com:mendersoftware/{{.Repo}}.git
+
+* Fetch the remote branches
+- git fetch open-source master
+- git fetch mender-test-bot {{.PRBranchName}}:localtmp
+
+* Checkout the localtmp branch
+- git checkout localtmp
+
+* Merge the Open-Source master branch into the PR branch
+- git merge open-source/master
```

Comment:
Is this right?

On conflicts, `mender-test-bot/{{.PRBranchName}}` is open-source based, right? So the procedure should be to check out that one and merge in `{{.Repo}}-enterprise/master`. At least that is how I recently fixed a conflict.

I don't think you need the `open-source` remote at all, and I think it would be best to avoid having it, to minimize the possibilities of a Git noob to push our enterprise code to the `open-source` remote.

Reply:
No, the `mender-test-bot` repos are the enterprise mirrors.

You're right though. We should merge with `enterprise` it seems. Then we can keep OS out of this wholly, which would be safest.

The master part is wrong though, isn't it? We need to merge feature branches, into feature branches. 

This is making me dizzy...

Reply:
> No, the `mender-test-bot` repos are the enterprise mirrors.

I meant the branch, the mergeosent_xx is OS absed.

> The master part is wrong though, isn't it? We need to merge feature branches, into feature branches.

Well, master when the PR is for master, release branch otherwise.